R/followers.R

```diff
@@ -7,12 +7,11 @@ friends <- function(user_id, n=NULL, ...) {
 }
 
 ffBase <- function(type, user_id, n=NULL, ...) {
-  if (can_access_other_account(user_id)) {
-    params <- parseUsers(user_id)
-    doCursorAPICall(paste(type, 'ids', sep='/'), 'ids', num=n, params=params, method='GET', ...)
-  } else {
-    warning("Can not lookup relationships for user id ", user_id)
+  if (!can_access_other_account(user_id)) {
+    warning("Cannot lookup relationships for user id ", user_id, ", query may fail!")
   }
+  params <- parseUsers(user_id)
+  doCursorAPICall(paste(type, 'ids', sep='/'), 'ids', num=n, params=params, method='GET', ...)
 }
```

Comment:
@geoffjentry What do you think about this? We get the warning if the oauth token doesn't have access to the user account, but attempt the query anyway in case it's an application token.